Starcounter.Uniform.Tests/Builder/UniDataTableTests.cs

```diff
@@ -0,0 +1,155 @@
+﻿using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using Starcounter.Uniform.Builder;
+using Starcounter.Uniform.Generic.FilterAndSort;
+using Starcounter.Uniform.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Starcounter.Uniform.Tests.Builder
+{
+    public class UniDataTableTests
```

Comment:
Tests that are also needed, but missing:

1. invalid input: order != "asc/desc", page index / size out of range
1. page changing behavior
1. reloading rows after changing filters / order

Reply:
I'm not saying all this should be in one class, but they all relate to `UniDataTable` class